internal/cli/internal/command/command.go

```diff
@@ -426,3 +426,18 @@ func LoadAppNameIfPresent(ctx context.Context) (context.Context, error) {
 
 	return app.WithName(ctx, name), nil
 }
+
+// RequireRegion is a Preparer which makes sure the user has selected a region via command line arguments,
+func RequireRegion(ctx context.Context) (context.Context, error) {
+	ctx, err := LoadAppConfigIfPresent(ctx)
```

Comment:
Why are we loading the application configuration here? `RequireRegion` doesn't have to do with app config, does it?